molpipeline/mol2mol/scaffolds.py

```diff
@@ -38,6 +46,37 @@ class MakeScaffoldGeneric(_MolToMolPipelineElement):
     Done to make scaffolds less speciffic.
     """
 
+    def __init__(
+        self,
+        generic_atoms: bool = False,
+        generic_bonds: bool = False,
+        name: str = "MakeScaffoldGeneric",
+        n_jobs: int = 1,
+        uuid: Optional[str] = None,
+    ) -> None:
+        """Initialize MakeScaffoldGeneric.
+
```

Comment:
Is there a specific example or an easy explanation of why you would want to use the generic_atoms/generic_bonds parameters? If you can think of something useful this would be a good addition to the doc-string here

Reply:
The current example is that for a given dataset I have multiple scaffolds, which are partially overlapping. I wanted to check how often the scaffold occurred as substructure, which is not working when all atoms are set to carbons and all bonds are single bonds.  
I added a short Note in the docu.